dotcom-rendering/src/model/enhanceCollections.ts

```diff
@@ -75,41 +75,38 @@ const enhanceCards = (
 	collections: FEFrontCard[],
 	containerPalette?: DCRContainerPalette,
 ): DCRFrontCard[] =>
-	collections
-		.filter(
-			(card: FEFrontCard): card is FEFrontCard & { format: CAPIFormat } =>
-				!!card.format,
-		)
-		.map((faciaCard, index) => {
-			const format = decideFormat(faciaCard.format);
-			const group = `${faciaCard.card.group}${
-				faciaCard.display.isBoosted ? '+' : ''
-			}`;
-			const dataLinkName = getDataLinkNameCard(format, group, index + 1);
-			return {
-				format,
-				dataLinkName,
-				url: faciaCard.header.url,
-				headline: faciaCard.header.headline,
-				trailText: faciaCard.card.trailText,
-				webPublicationDate: faciaCard.card.webPublicationDateOption
-					? new Date(
-							faciaCard.card.webPublicationDateOption,
-					  ).toISOString()
-					: undefined,
-				image: faciaCard.properties.maybeContent?.trail.trailPicture
-					?.allImages[0].url,
-				kickerText:
-					faciaCard.header.kicker?.item?.properties.kickerText,
-				supportingContent: faciaCard.supportingContent
-					? enhanceSupportingContent(
-							faciaCard.supportingContent,
-							format,
-							containerPalette,
-					  )
-					: undefined,
-			};
-		});
+	collections.map((faciaCard, index) => {
+		// Cards with snaps won't have a set format.
```

Comment:
This comment is perhaps a bit confusing out of context. What is it referring to?